.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,31 @@
+name: Bug Report
+description: Create a bug report to help us fix issues with the Quilt website.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this bug! If you need real-time help, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: "Describe the bug as clearly as possible. The more detail you provide means the sooner the bug can be fixed! This includes: steps to reproduce or situation you encountered the bug in, or any additional context that could be helpful."
+    validations:
+      required: true
+
+  - type: dropdown
```

Comment:
Please make this a free-text field, and add an optional question for the version of the browser if they know it.

It would also be useful to know if the user has tried in a different browser, and what device and operating system they're using (to know if the issue is mobile-specific, for example).